tests/queries/0_stateless/03622_max_temporary_data_size_on_disk.config.xml

```diff
@@ -0,0 +1,35 @@
+<clickhouse>
```

Comment:
Perhaps it would be better to add this test case to the integration test suite at https://github.com/ClickHouse/ClickHouse/tree/master/tests/integration/test_temporary_data rather than as a stateless test that runs a separate server. However, this is fine for now - we can refactor it in a separate PR if we decide it's necessary.